mflowgen/Tile_MemCore/construct.py

```diff
@@ -230,20 +253,40 @@ def construct():
   g.connect_by_name( drc,      debugcalibre )
   g.connect_by_name( lvs,      debugcalibre )
 
+  # Pwr aware steps:
+  if pwr_aware:
+      g.connect_by_name( power_domains,        dc           )
+      g.connect_by_name( power_domains,        init         )
+      g.connect_by_name( power_domains,        power        )
+      g.connect_by_name( power_domains,        place        )
+      g.connect_by_name( power_domains,        cts          )
+      g.connect_by_name( power_domains,        postcts_hold )
+      g.connect_by_name( power_domains,        route        )
+      g.connect_by_name( power_domains,        postroute    )
+      g.connect_by_name( power_domains,        signoff      )
+      #g.connect(power_domains.o('pd-globalnetconnect.tcl'), power.i('globalnetconnect.tcl'))
+
   #-----------------------------------------------------------------------
   # Parameterize
   #-----------------------------------------------------------------------
 
   g.update_params( parameters )
 
+  # Update PWR_AWARE variable
+  if pwr_aware:
+      dc.update_params( { 'PWR_AWARE': parameters['PWR_AWARE'] }, True )
+      init.update_params( { 'PWR_AWARE': parameters['PWR_AWARE'] }, True )
+      power.update_params( { 'PWR_AWARE': parameters['PWR_AWARE'] }, True )
```

Comment:
@ctorng they do... True is the "allow_new" setting, which is required since PWR_AWARE is not a default parameter for these steps. The problem here is that this code is only executed when pwr_aware is true. It should always be executed.